_emails/redwoodcity.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Redwood City, California
+permalink: "/redwoodcity"
+name: Letter to the Mayor, Vice Mayor, and City Council Members
+city: Redwood City
+state: CA
+recipients:
+- council@redwoodcity.org
+- dhoward@redwoodcity.org
+- smasur@redwoodcity.org
+- aaguirre@redwoodcity.org
+- ibain@redwoodcity.org
+- jborgens@redwoodcity.org
+- ghale@redwoodcity.org
+- dreddy@redwoodcity.org
+subject: Redwood City Resident for Defunding the RCPD
+body: |-
+	To Mayor Howard, Vice Mayor Masur, and Redwood City City Council Members,
+
+	My name is \[YOUR NAME\], and I am a resident of Redwood City. This past week, our nation and community have been gripped by protests calling for an end to racism and anti-blackness and a complete overhaul in our approach to criminal justice in America. We are demanding that our voices be heard, and that real change be made to the way this city allocates its resources.
+
+	The Redwood City Police Department is allocated $46.5M (33.9%) of the city’s FY 2019-20 operating budget and $48.9M (34.3%) of the city's recommended FY 2020-21 operating budget. It is shameful and unproductive that, in comparison, in the recommended FY 2020-21 operating budget only allocates $6.8M to affordable housing. Research shows that a living wage, access to holistic health services and treatment including mental health services, educational opportunity, and stable housing are far more successful at reducing crime than punitive systems like police or prisons (Source: Popular Democracy). 
+
+	There is not just a need for police reform, but police defunding and abolition, because the entire structure of a punitive police force is inherently corrupt and ineffective. The average police recruit spends 58 hours learning how to shoot and only 8 hours learning how to de-escalate (Source: Campaign Zero). They are not trained or equipped to react to the vast majority of crises. In the journal of Sociology of Race and Ethnicity, Phillip McHarris (PhD candidate at Yale focusing on race) argues that we must work towards a reality in which healthcare workers and emergency response teams handle substance abuse, domestic violence, homelessness, or mental health cases. Policies to “improve the police” are not enough, as there’s no evidence that implicit bias training or community relations initiatives help with reducing the abuses of policing (The Nation, The Atlantic). We need to reimagine public safety to prioritize alternatives to conflict rather than defaulting to violence.
```

Comment:
```suggestion
	There is not a need for police reform, but police defunding and abolition, because the entire structure of a punitive police force is inherently corrupt and ineffective. The average police recruit spends 58 hours learning how to shoot and only 8 hours learning how to de-escalate (Source: Campaign Zero). They are not trained or equipped to react to the vast majority of crises. In the journal of Sociology of Race and Ethnicity, Phillip McHarris (PhD candidate at Yale focusing on race) argues that we must work towards a reality in which healthcare workers and emergency response teams handle substance abuse, domestic violence, homelessness, or mental health cases. Policies to “improve the police” are not enough, as there’s no evidence that implicit bias training or community relations initiatives help with reducing the abuses of policing (The Nation, The Atlantic). We need to reimagine public safety to prioritize alternatives to conflict rather than defaulting to violence.
```

Reply:
Clarified, thank you!